server/runtime/src/main/resources/infinispan-defaults.xml

```diff
@@ -25,6 +25,11 @@
       <distributed-cache-configuration name="org.infinispan.DIST_SYNC" remote-timeout="17500" configuration="org.infinispan.LOCAL">
          <state-transfer timeout="60000"/>
       </distributed-cache-configuration>
+      <distributed-cache-configuration name="example.PROTOBUF_DIST" remote-timeout="17500" configuration="org.infinispan.LOCAL">
```

Comment:
Is ```"example.PROTOBUF_DIST"``` the first of this kind? Are we settled on using the ```example```  prefix for these? any other example of ```example```?

Reply:
https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/server.20templates

Reply:
:+1: